hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -194,16 +194,16 @@ If self update is enabled then the bootloader will look for (pieeprom.upd + piee
 
 Since the updates files are in the same format as generated by rpi-eeprom-update you can use rpi-eeprom-update to install the files to /boot so long as /boot mounts the approprate boot device / tftp-root (via NFS).
 
-**The bootloader only reads the configuration from the bootconf.txt in the EEPROM and not the one in the update files. Therefore, in order to enable SELF_UPDATE you have to first update the bootloader via the SD-CARD or FLASHROM.**
+**The bootloader only reads the configuration from the bootconf.txt in the EEPROM and not the one in the update files. Therefore, in order to enable ``SELF_UPDATE`` you have to first update the bootloader via the SD-CARD or FLASHROM.**
 
 Default: 0  
 Version: pieeprom-2020-04-16.bin  
 
 ``pieeprom-2020-05-15.bin - BETA``
-SELF_UPDATE and bootloader_update are now enabled by default so that rpi-update will work by default. Setting either of these parameters to zero prevents self-updates.
+The ``SELF_UPDATE`` EEPROM property and ``bootloader_update`` config.txt property are now enabled by default so that ``rpi-eeprom-update`` may be used without requiring extra customization. Setting either of these parameters to zero prevents self-updates.
 
```

Comment:
You are going to hate me for this, but you only need one backtick `fred`...

Reply:
changed although both seem to be supported